napari/components/viewer_model.py

```diff
@@ -70,7 +70,6 @@ def __init__(self, title='napari', ndisplay=2, order=(), axis_labels=()):
             help=Event,
             title=Event,
             reset_view=Event,
-            active_layer=Event,
```

Comment:
I do not like this. This is removing without even a single version for a change. 
From my point of view, it is one of the most important signals for persons who write their own widgets which depends on the active layer. I do not remember who may use this but I call @kevinyamauchi @brisvag @alisterburt (I have hope that I found proper persons)

Reply:
as of right now we aren't using this (which doesn't invalidate the point about deprecation, difficult to find the balance for sure)

Reply:
The current event deprecation syntax doesn't allow us to deprecated an event without proving a direct replacement on the object, something we aren't doing here. I'm trying to get it to work, but it will require a big change to how we deprecate events